internal/sys/map_flags.go

```diff
@@ -0,0 +1,13 @@
+package sys
+
+import "github.com/cilium/ebpf/internal/unix"
+
+type MapFlags uint32
```

Comment:
Please update `gentypes` so that https://github.com/cilium/ebpf/blob/c662d71024c8d86e1b365ada06d66fcfd8ef3905/internal/sys/types.go#L683 uses MapFlags as well. The setup for that happens here: https://github.com/cilium/ebpf/blob/c662d71024c8d86e1b365ada06d66fcfd8ef3905/internal/cmd/gentypes/main.go#L64-L86
There might be other structs that need the same change, please take a look.

Reply:
As yes, I made the change for `sys.MapInfo` and `sys.MapCreateAttr`